app/views/hyrax/admin/users/index.html.erb

```diff
@@ -0,0 +1,46 @@
+<% content_for :page_header do %>
+  <h1><span class="fa fa-user"></span> <%= t('hyrax.admin.users.index.title') %></h1>
+<% end %>
+
+<div class="panel panel-default users-listing">
+  <div class="panel-heading">
+      <%= t('hyrax.admin.users.index.describe_users_html', count: @presenter.user_count) %>
+  </div>
+
+  <div class="panel-body">
+    <div class="table-responsive">
+      <table class="table table-striped datatable">
+        <thead>
+          <tr>
+            <th></th>
+            <th><%= t('.id_label') %></th>
+            <th><%= t('.role_label') %></th>
+            <th><%= t('.access_label') %></th>
+          </tr>
+        </thead>
+        <tbody>
+          <% @presenter.users.each do |user| %>
+            <tr>
+              <td><%= link_to hyrax.profile_path(user) do %>
+                    <%= image_tag(user.avatar.url(:thumb), width: 30) if user.avatar.file %>
+                  <% end %>
+              </td>
+              <br>
+              <td><%= link_to user.email, hyrax.profile_path(user) %></td>
+              <td><% roles = @presenter.user_roles(user) %>
+                  <ul><% roles.each do |role| %>
+                    <li><%= role %></li>
+                    <% end %>
+                  </ul>
+              </td>
+              <td>
+                <%# in the case that a user is created who never signs in, this is necessary %>
+                <%= time_ago_in_words(user.last_sign_in_at || user.created_at) %>
```

Comment:
@jcoyne "I'd like to" sounds like a personal preference, but you have given no explanation as to why this is a better method.  I willingly admit that my knowledge of javascript is quite limited, and I am very interested to understand why you believe the .js method is better.

From what I see, the ruby method is currently a better option. The method time_ago_in_words is used elsewhere in the codebase, so using it and adding translations for it will allow for many of the translations to be handled immediately.  I could only find time-elements.js used in one place (did I miss some?).  Using it in additional places before it can handle translations seems a bit premature in my view. 

A better option might be to create an issue to add the translation capability to time-elements AND change all uses of time_ago_in_words to use time-elements at the same time, once those translations are available.